src/platform/hashtable.c

```diff
@@ -841,7 +841,7 @@ Return Value:
         Signature = CXPLAT_HASH_ALT_SIGNATURE;
     }
 
-    CXPLAT_HASHTABLE_LOOKUP_CONTEXT LocalContext;
+    CXPLAT_HASHTABLE_LOOKUP_CONTEXT LocalContext = {0};
```

Comment:
I don't think this one is truly necessary. Also, since this is in the hot path, I'm inclined to ignore this one, rather than this change. Thoughts?